src/contracts/services/moneyMarket.ts

```diff
@@ -0,0 +1,582 @@
+  lcdEndpoint?: string,
+  chainId?: string,
+  pagination?: Pagination,
+}) => getActiveQueryClient$(lcdEndpoint, chainId).pipe(
+  switchMap(({ client }) => sendSecretClientContractQuery$({
+    queryMsg: msgQueryMoneyMarketMarkets(pagination),
+    client,
+    contractAddress,
+    codeHash,
+  })),
+  map((response) => parseMoneyMarketGetMarkets(response as GetMarketsResponse)),
+  first(),
+);
+
+/**
+ * query the money market config
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+async function queryMoneyMarketConfig({
+  contractAddress,
+  codeHash,
+  lcdEndpoint,
+  chainId,
+}: {
+  contractAddress: string,
+  codeHash?: string,
+  lcdEndpoint?: string,
+  chainId?: string,
+}) {
+  return lastValueFrom(queryMoneyMarketConfig$({
+    contractAddress,
+    codeHash,
+    lcdEndpoint,
+    chainId,
+  }));
+}
+
+/**
+ * query the money market get markets query
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+async function queryMoneyMarketGetMarkets({
+  contractAddress,
+  codeHash,
+  lcdEndpoint,
+  chainId,
+  pageSize,
+  page,
+}: {
+  contractAddress: string,
+  codeHash?: string,
+  lcdEndpoint?: string,
+  chainId?: string,
+  pageSize?: number,
+  page?: number,
+}) {
+  return lastValueFrom(queryMoneyMarketGetMarkets$({
+    contractAddress,
+    codeHash,
+    lcdEndpoint,
+    chainId,
+    pagination: pageSize !== undefined && page !== undefined ? {
+      page_size: pageSize,
+      page,
+    } : undefined,
+  }));
+}
+
+/**
+ * query the money market get collateral query
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+async function queryMoneyMarketGetCollateral({
+  contractAddress,
+  codeHash,
+  lcdEndpoint,
+  chainId,
+  pageSize,
+  page,
+}: {
+  contractAddress: string,
+  codeHash?: string,
+  lcdEndpoint?: string,
+  chainId?: string,
+  pageSize?: number,
+  page?: number,
+}) {
+  return lastValueFrom(queryMoneyMarketGetCollateral$({
+    contractAddress,
+    codeHash,
+    lcdEndpoint,
+    chainId,
+    pagination: pageSize !== undefined && page !== undefined ? {
+      page_size: pageSize,
+      page,
+    } : undefined,
+  }));
+}
+
+/**
+ * parses the config reponse from a batch query of
+ * multiple money market contracts
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+const parseBatchQueryMoneyMarketConfig = (
+  response: BatchQueryParsedResponse,
+): BatchMoneyMarketConfigs => response.map((item) => ({
+  moneyMarketContractAddress: item.id as string,
+  config: parseMoneyMarketConfig(item.response),
+  blockHeight: item.blockHeight,
+}));
+
+/**
+ * query the config for money market contracts at one time
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+function batchQueryMoneyMarketConfig$({
+  queryRouterContractAddress,
+  queryRouterCodeHash,
+  lcdEndpoint,
+  chainId,
+  moneyMarketContracts,
+  batchSize = SERVICE_BATCH_SIZE.PAIR_INFO,
```

Comment:
would just leave this empty for now so we don't accidentally forget about this and leave pair_info in as the default.